controller_cat.go

```diff
@@ -138,16 +134,16 @@ func (c *controller) asyncPeerRequest(peer *Peer) ([]Endpoint, error) {
 		c.shareMtx.Unlock()
 	}()
 
-	req := newParcel(TypePeerRequest, []byte("Peer Request"))
+	req := newParcel(TypePeerRequest, nil)
```

Comment:
The p2p system doesn't allow for parcels with a zero-length payload: https://github.com/WhoSoup/factom-p2p/blob/42ae708d267cd5d547138a04c91f9a2d5c51fdd6/parcel.go#L72-L74
This eliminates a lot of further-down-the-road checks inside the library, and since the Payload is passed directly to the application for some parcels, I thought it safer to just prevent potential nil pointers going where nil pointers shouldn't go

The payload being "Peer Request" is for backward compatibility with V9